src/v/storage/types.h

```diff
@@ -360,7 +357,6 @@ struct local_log_reader_config {
 
     model::offset start_offset;
     model::offset max_offset;
-    size_t min_bytes;
```

Comment:
I can't remember why min bytes existed, except perhaps as a convenience. But I suppose now we just rebuild readers and retry.